CONTRIBUTING.md

```diff
@@ -6,43 +6,57 @@ happen through Git and Pull Requests.
 
 ## Requirements
 
-To contribute to ECS the following tools are expected to be running on your machine:
+You need these tools to contribute to ECS:
 
 * [Git](https://git-scm.com/)
 * [Python](https://www.python.org/)
 * [Go](https://golang.org/)
 
 ## Steps to contribute
 
-To contribute changes to ECS follow the steps below:
-
-* Create [your own fork](https://help.github.com/articles/fork-a-repo/) of the ECS repo.
-* Clone your fork to your machine.
-* Run `git checkout -b branch-name` and replace `branch-name` according to your change
-* Apply your changes to the `.yml` files as needed
-* Run `make` to update generated files like `schema.csv` and `schema.md`
-* Run `git commit -a -m "your message"`
-* Run `git push --set-upstream origin branch-name` (assumes your clone remote is called `origin`)
-* Create a [Pull Request](https://help.github.com/articles/creating-a-pull-request/) against the ECS repo. (if you go to the website, it should pop up as a button with your branch directly)
-* Wait for reviews on your PR and collaborate.
-
-Notes: Make sure to always only push changes against your own fork.
-
+Here are the steps for contributing to ECS.
+
+* Set up your git environment.
+  - Create [your own fork](https://help.github.com/articles/fork-a-repo/) of the ECS repo.
+  - Clone your fork to your machine.
+* Create a working branch on your machine.
+  - Run `git checkout -b branch-name`, where `branch-name` is the name you want to give your local branch
+* Do your work. 
+  - Make changes to the `.yml` files as needed
+* Run `make` to update generated files such as `schema.csv` and `schema.md`
```

Comment:
There is no `schema.md` anymore. Perhaps we can just mention here that this will update the generated files.

This is probably also a good place to mention that the README is fully generated and data has to changed under `docs`.